packages/graphql/src/schema/scalars/Date.ts

```diff
@@ -17,16 +17,28 @@
  * limitations under the License.
  */
 
-import { GraphQLScalarType } from "graphql";
-import neo4j from "neo4j-driver";
+import { GraphQLError, GraphQLScalarType } from "graphql";
+import neo4j, { Date as Neo4jDate, isDate } from "neo4j-driver";
```

Comment:
One of two points of contention here as unsure if this will work with older driver versions. If not, we need to either update our minimum, or find an alternative route.